vividus-http-client/src/main/java/org/vividus/http/context/CacheableClientCredentialsHttpClientContextFactory.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2019-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.http.context;
+
+import org.apache.http.HttpHost;
+import org.apache.http.client.AuthCache;
+import org.apache.http.client.CredentialsProvider;
+import org.apache.http.client.protocol.HttpClientContext;
+import org.apache.http.impl.auth.BasicScheme;
+import org.apache.http.impl.client.BasicAuthCache;
+import org.vividus.http.client.ClientBuilderUtils;
+
+public class CacheableClientCredentialsHttpClientContextFactory implements HttpClientContextFactory
```

Comment:
Shouldn't it be like Cach`ING`ClientCredentialsContextFactory
I believe able for the interfaces :)

Reply:
![image](https://user-images.githubusercontent.com/17816729/93616806-e1d2d200-f9dd-11ea-8ecf-4e688e01e9a8.png)